GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/Gas.cs

```diff
@@ -73,7 +72,7 @@ public void SetGasLevelHelper(Image image, GameObject obj)
         // set color change according to the bar length
         if (_gasLevel < 0.2 * _fullGasLevel)
         {
-            image.color = new Color32(196,92,29,255 );
+            image.color = new Color32(196,92,29,255);
         } else if (_gasLevel < 0.4 * _fullGasLevel)
```

Comment:
_fullGasLevel and some others can be defined as const

Reply:
I will do that